pybossa/cache/projects.py

```diff
@@ -206,7 +205,7 @@ def get_all_featured(category=None):
     sql = text('''SELECT project.id, project.name, project.short_name, project.info,
                project.created, project.updated, project.description,
                "user".fullname AS owner FROM project, "user"
-               WHERE project.featured=true AND project.hidden=0
+               WHERE project.featured=true
```

Comment:
I know that if a project is featured, it should be published. Can we include it here on the query?